src/main/kotlin/org/rust/lang/core/psi/ext/RsAttr.kt

```diff
@@ -8,15 +8,86 @@ package org.rust.lang.core.psi.ext
 import org.rust.lang.core.psi.RsInnerAttr
 import org.rust.lang.core.psi.RsMetaItem
 import org.rust.lang.core.psi.RsOuterAttr
+import org.rust.lang.core.resolve.MACROS
 
 interface RsAttr : RsElement {
     val metaItem: RsMetaItem
 }
 
+/**
+ * Source of attributes: [https://doc.rust-lang.org/1.44.1/reference/attributes.html#built-in-attributes-index]
```

Comment:
I updated the list from `1.30.0` to `1.44.1` and removed some names that no longer exist in the new version.